src/rmodels.c

```diff
@@ -4864,7 +4864,9 @@ static BoneInfo *LoadBoneInfoGLTF(cgltf_skin skin, int *boneCount)
     for (unsigned int i = 0; i < skin.joints_count; i++)
     {
         cgltf_node node = *skin.joints[i];
-        strncpy(bones[i].name, node.name, sizeof(bones[i].name));
+        if (node.name != NULL) {
```

Comment:
Please, could you avoid the brackets, in raylib one line statements use no-brackets:
```c
if (node.name != NULL) strncpy(bones[i].name, node.name, sizeof(bones[i].name));
```